app/api/helpers/tasks.py

```diff
@@ -663,3 +662,12 @@ def delete_translations(self, zip_file_path):
         os.remove(zip_file_path)
     except:
         logger.exception('Error while deleting translations zip file')
+
+
+@celery.task(name='rename.chat.room')
+def rename_chat_room(event_id):
+    event = db.session.query(Event).filter_by(id=int(event_id)).first()
+    try:
+        rename_rocketchat_room(event=event)
+    except:
```

Comment:
Never catch bare exception. Better let it throw so we atleast know in stacktrace why it failed